docs/components/compact.md

```diff
@@ -101,12 +101,12 @@ There can be few valid use cases for vertical compaction:
 * Races between multiple compactions, for example multiple compactors or between compactor and Prometheus compactions. While this will have extra
 computation overhead for Compactor it's safe to enable vertical compaction for this case.
 * Backfilling. If you want to add blocks of data to any stream where there is existing data already there for the time range, you will need enabled vertical compaction.
-* Offline deduplication of series. It's very common to have the same data replicated into multiple streams. We can distinguish two common series duplications, `one-to-one` and `realistic`:
-  * `one-to-one` duplication is when same series (series with the same labels from different blocks) for the same range have **exactly** the same samples: Same values and timestamps.
+* Offline deduplication of series. It's very common to have the same data replicated into multiple streams. We can distinguish two common series deduplications, `one-to-one` and `penalty`:
+  * `one-to-one` deduplication is when same series (series with the same labels from different blocks) for the same range have **exactly** the same samples: Same values and timestamps.
 This is very common while using [Receivers](../components/receive.md) with replication greater than 1 as receiver replication copies exactly the same timestamps and values to different receive instances.
-  * `realistic` duplication is when same series data is **logically duplicated**. For example, it comes from the same application, but scraped by two different Prometheus-es. Ideally
+  * `penalty` deduplication is when same series data is **logically duplicated**. For example, it comes from the same application, but scraped by two different Prometheus-es. Ideally
 this requires more complex deduplication algorithms. For example one that is used to [deduplicate on the fly on the Querier](query.md#run-time-deduplication-of-ha-groups). This is common
-case when Prometheus HA replicas are used. [Offline deduplication for this is in progress](https://github.com/thanos-io/thanos/issues/1014).
+case when Prometheus HA replicas are used. You can enable this deduplication via `--deduplication.func=penalty` flag.
```

Comment:
I am not sure about the 1% case. I only know it is related to counter reset/ rate functions. Is this https://github.com/thanos-io/thanos/issues/2890 the right issue for it? 

@bwplotka @kakkoyun for more input.

Reply:
Maybe let's say something like:

```
even though the deduplication algorithm has been battle-tested for quite a long time but still very few issues come up from time to time such as https://github.com/thanos-io/thanos/issues/2890
```

? It would be more concrete. And it wouldn't sound like we know that it is bad in some 1% of cases & we aren't fixing it :smile:

Reply:
Updated docs. PTAL